src/tests/system/tests/test_sss_override.py

```diff
@@ -0,0 +1,274 @@
+from __future__ import annotations
+
+import pytest
+from sssd_test_framework.roles.client import Client
+from sssd_test_framework.roles.generic import GenericProvider
+from sssd_test_framework.topology import KnownTopology
+
+
+@pytest.mark.importance("high")
+@pytest.mark.sss_override
```

Comment:
This marker is highly redundant and completely unnecessary. You can already select these tests in multiple way even without the marker:

```
pytest --mh-config=./mhc.yaml --mh-lazy-ssh --collect-only -k sss_override
pytest --mh-config=./mhc.yaml --mh-lazy-ssh --collect-only -k sss_overrides
pytest --mh-config=./mhc.yaml --mh-lazy-ssh --collect-only -k override
pytest --mh-config=./mhc.yaml --mh-lazy-ssh --collect-only -k overrides
pytest --mh-config=./mhc.yaml --mh-lazy-ssh --collect-only -k test_sss_overrides
pytest --mh-config=./mhc.yaml --mh-lazy-ssh --collect-only -k override
pytest --mh-config=./mhc.yaml --mh-lazy-ssh --collect-only tests/test_sss_override.py
pytest --mh-config=./mhc.yaml --mh-lazy-ssh --collect-only -k test_sss_override.py
```

You choose. The marker does not add anything new. The way how pytest filtering works is that pytest creates set of keyword out of the test, class, module names and attached markers. So as long as the test already contains "sss_override" in it, the marker does not add any new keyword to this set.

Reply:
Names should be used to identify the test group as much as possible. Additional markers can be use to group tests in multiple modules if needed. Such as mark "test_gpo" with "access_provider". But ideally, the test should be named "test_access_gpo" and the marker is not needed again.

Reply:
Ack, I understand how it's going to be used now. Removing the marker.